docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -0,0 +1,291 @@
+|**`dac_cert`**|`(DAC) Device Attestation Certificate`|<1, 1204> B|*hex string*|mandatory|The Device Attestation Certificate (DAC) and corresponding private key, are unique to each Matter Device. The DAC is used for the Device Attestation process, and to perform commissioning into a Fabric. The DAC is a DER-encoded X.509v3-compliant certificate as defined in RFC 5280.|
+|**`dac_key`**|`DAC private key`|68 B|*hex string*|mandatory|The Private key associated with the Device Attestation Certificate (DAC). This key should be encrypted, and maximum security should be guaranteed while generating and providing it to factory data.|
+|**`pai_cert`**|`Product Attestation Intermediate`|<1, 1204> B|*hex string*|mandatory|An intermediate certificate is an X.509 certificate, which has been signed by the root certificate. The last intermediate certificate in a chain is used to sign the leaf (the Matter device) certificate.|
+|**`spake2_it`**|`SPAKE2 Iteration Counter`|4 B|*uint32*|mandatory|The Spake2 iteration count is associated with the ephemeral PAKE passcode verifier to be used for the commissioning. The iteration count is used as a crypto parameter to process spake2 verifier.|
+|**`spake2_salt`**|`SPAKE2 Salt`| <36, 68> B|*hex string*|mandatory|The spake2 salt is random data that is used as an additional input  to a one-way function that “hashes” data. A new salt should be randomly generated for each password.|
+|**`spake2_verifier`**|`SPAKE2 Verifier`| 97 B|*hex string*|mandatory|The spake 2 verifier generated using default SPAKE2 salt, iteration count and passcode. This value can be used for development or testing purposes.|
+|**`discriminator`**|`Discriminator`| 2 B|*uint16*|mandatory|A 12-bit value matching the field of the same name in the setup code. Discriminator is used during a discovery process.|
+|**`passcode`**|`SPAKE passcode`|4 B|*uint32*|optional|A pairing passcode is a 27-bit unsigned integer which serves as a proof of possession during commissioning. Its value shall be restricted to the values 0x0000001 to 0x5F5E0FE (00000001 to 99999998 in decimal), excluding the invalid Passcode values: 00000000, 11111111, 22222222, 33333333, 44444444, 55555555, 66666666, 77777777, 88888888, 99999999, 12345678, 87654321|
+|**`user`**|`User data`| variable|*JSON string*|max 1024 B|The user data is provided in JSON format. This parameter is optional and depends on user/manufacturer purpose. It is getting as a string from persistent storage and then should be parsed in a user application. This data is not used in the Matter stack.|
+
+### Factory data format
+
+The Factory data set is represented by JSON format that is regulated by [JSON Schema](https://github.com/project-chip/connectedhomeip/blob/master/scripts/tools/nrfconnect/nrfconnect_factory_data.schema) file. All parameters are divided between `mandatory `and `optional.` 
```

Comment:
I would this section to describe the factory data format on a device, not only its intermediate representation. I think it's worth adding a sentence that the factory data is stored on a device in the CBOR format, but the JSON format is used as an intermediate, human-readable representation of the data.